odh-model-controller/controllers/inferenceservice_controller.go

```diff
@@ -74,11 +74,6 @@ func (r *OpenshiftInferenceServiceReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, err
 	}
 
-	err = r.ReconcileNamespace(inferenceservice, ctx)
```

Comment:
Any reason to keep [inferenceservice_namespace.go](https://github.com/opendatahub-io/modelmesh-serving/blob/main/odh-model-controller/controllers/inferenceservice_namespace.go) around if this is being removed? This seems like its only usage.

Reply:
I did it remove it as part of the PR:
![Uploading image.png…]()